### PR TITLE
[fix] #2880: Fix block streaming

### LINKED ISSUE #2880: [BUG] Block Streaming API doesn't work as a stream
### GIT commit hash

b783f10fa7de26ed1fdd4c526bd162f8636f1a65

### Minimum working example

- Start Iroha with any configuration, peers amount and genesis
- Listen for Blocks Stream via any existing client
- Inspect incoming blocks

### Expected behaviour

I expected to see how blocks are sent from Iroha on-the-fly when they appear in the blockchain.

### Actual behaviour

I got all existing blocks Iroha had (depending on the `height` param I've provided on the handshake) and then Iroha halted the connection.

[Spec](https://github.com/appetrosyan/iroha/blob/iroha2-staging/docs/source/references/api_spec.md#blocks-stream) says:

> Via this endpoint client first provides the starting block number(i.e. height) in the subscription request. After sending the confirmation message, server starts streaming all the blocks from the given block number up to the current block and continues to stream blocks as they are added to the blockchain.

### Operating system

Both on local macOS 12.2.1 and in a docker image you can explore [here](https://github.com/hyperledger/iroha-javascript/blob/f90aaffaeae9f2332b2a567b54f3f7cb879ec7e8/Jenkinsfile) 

### LibC type and version

On macOS I can't determine; what is within the docker image - same

### Current environment

Source code build

### Logs in JSON format

<details>
  <summary>Log contents (log from the automated test)</summary>
  
  ```
[2022-10-18T13:53:21.026Z] 2022-10-18T13:53:20.958Z @iroha2/test-peer:stdout 2022-10-18T13:53:20.958135Z  INFO iroha: Hyperledgerいろは2にようこそ！
[2022-10-18T13:53:21.026Z] 2022-10-18T13:53:20.958Z @iroha2/test-peer:stdout 2022-10-18T13:53:20.958155Z  INFO iroha: (translation) Welcome to Hyperledger Iroha 2!
[2022-10-18T13:53:21.026Z] 2022-10-18T13:53:20.958Z @iroha2/test-peer:stdout 2022-10-18T13:53:20.958815Z  INFO iroha: Starting peer listen_addr=127.0.0.1:1337
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:20.958Z @iroha2/test-peer:stdout 2022-10-18T13:53:20.958831Z  INFO iroha_p2p::network: Binding listener listen_addr=127.0.0.1:1337
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:20.958Z @iroha2/test-peer:stdout 2022-10-18T13:53:20.958866Z  INFO iroha_p2p::network: Starting network actor listen_addr=127.0.0.1:1337
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:20.961Z @iroha2/test-peer:stdout 2022-10-18T13:53:20.961077Z  INFO iroha_core::kura: Loaded 0 blocks at init.
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:20.962Z @iroha2/test-peer:stdout 2022-10-18T13:53:20.962445Z ERROR iroha: Telemetry did not start
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:20.962Z @iroha2/test-peer:stdout 2022-10-18T13:53:20.962772Z  INFO iroha: Starting Iroha
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:21.007Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.007667Z  INFO request{method=GET path=/health version=HTTP/1.1 remote.addr=127.0.0.1:55636}: warp::filters::trace: processing request
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:21.007Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.007739Z  INFO request{method=GET path=/health version=HTTP/1.1 remote.addr=127.0.0.1:55636}: warp::filters::trace: finished processing with success status=200
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:21.009Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.009093Z  INFO request{method=GET path=/status version=HTTP/1.1 remote.addr=127.0.0.1:42540}: warp::filters::trace: processing request
[2022-10-18T13:53:21.027Z] 2022-10-18T13:53:21.009Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.009136Z  INFO request{method=GET path=/status version=HTTP/1.1 remote.addr=127.0.0.1:42540}: warp::filters::trace: finished processing with success status=200
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.212Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.212836Z  INFO run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Initializing iroha using the genesis block.
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.213Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.212889Z  INFO run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Publishing genesis block. block_hash=e70e18a61d4fc0cd66fba31b8a0985ff881cbaafea0925bafacfebfc5dbf88c4
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.213Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.213303Z  INFO run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Created a block to commit. peer_role=Leader block_hash=2348e1537306cb3ab812c22a85c74cc6130ffc3fd8683b9428d6e01a442ac8da
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.213Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.213755Z ERROR run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Some events failed to be sent e=channel closed
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.213Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.213775Z ERROR run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Some events failed to be sent e=channel closed
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.213Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.213798Z  INFO run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Committing block prev_peer_role=Leader new_peer_role=Leader new_block_height=1 block_hash=2348e1537306cb3ab812c22a85c74cc6130ffc3fd8683b9428d6e01a442ac8da
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.259Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.259760Z  INFO request{method=GET path=/status version=HTTP/1.1 remote.addr=127.0.0.1:42542}: warp::filters::trace: processing request
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.259Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.259828Z  INFO request{method=GET path=/status version=HTTP/1.1 remote.addr=127.0.0.1:42542}: warp::filters::trace: finished processing with success status=200
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.261Z @iroha2/client:blocks-stream:websocket opening connection to 'ws://127.0.0.1:8080/block/stream'
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.262Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.262124Z  INFO request{method=GET path=/block/stream version=HTTP/1.1 remote.addr=127.0.0.1:55642}: warp::filters::trace: processing request
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.262Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.262197Z  INFO request{method=GET path=/block/stream version=HTTP/1.1 remote.addr=127.0.0.1:55642}: warp::filters::trace: finished processing with status status=101 error=None
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.262Z @iroha2/client:blocks-stream:websocket connection opened
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.262Z @iroha2/client:blocks-stream:websocket message Uint8Array(2) [ 1, 0 ]
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.263Z @iroha2/client:blocks-stream subscription accepted
[2022-10-18T13:53:21.282Z] 2022-10-18T13:53:21.264Z @iroha2/client:blocks-stream:websocket message Uint8Array(1957) [
[2022-10-18T13:53:21.282Z]     1,   1,   1,  60, 201,  94, 235, 131,   1,   0,   0,   0,
[2022-10-18T13:53:21.282Z]     0,   0,   0,   0,   0,   0,   0, 208,   7,   0,   0,   0,
[2022-10-18T13:53:21.282Z]     0,   0,   0,   1,   0,   0,   0,   0,   0,   0,   0,   0,
[2022-10-18T13:53:21.282Z]     0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,
[2022-10-18T13:53:21.282Z]     0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,
[2022-10-18T13:53:21.282Z]     0,   0,   0,   0,   0,   0,   0,  49,  86, 120, 244, 149,
[2022-10-18T13:53:21.282Z]    19, 148, 118, 168, 159, 206, 242, 128, 104,  40, 131,  22,
[2022-10-18T13:53:21.282Z]    95, 111,  82, 246, 183,  74, 141,  34,  87,  39,  88, 244,
[2022-10-18T13:53:21.282Z]   133, 165,  93,   0,
[2022-10-18T13:53:21.282Z]   ... 1857 more items
[2022-10-18T13:53:21.282Z] ]
[2022-10-18T13:53:21.283Z] 2022-10-18T13:53:21.267Z @iroha2/client:blocks-stream new block: Enum { tag: 'V1', value: { header: { timestamp: 1666101201212n, consensus_estimation: 2000n, height: 1n, previous_block_hash: [Uint8Array], transactions_hash: [Uint8Array], rejected_transactions_hash: [Uint8Array], genesis_topology: [Enum] }, rejected_transactions: [], transactions: [ [Enum] ], event_recommendations: [], signatures: { signatures: [Map] } } }
[2022-10-18T13:53:21.283Z] 2022-10-18T13:53:21.277Z @iroha2/client:blocks-stream:websocket message Uint8Array(1957) [
[2022-10-18T13:53:21.283Z]     1,   1,   1,  60, 201,  94, 235, 131,   1,   0,   0,   0,
[2022-10-18T13:53:21.283Z]     0,   0,   0,   0,   0,   0,   0, 208,   7,   0,   0,   0,
[2022-10-18T13:53:21.283Z]     0,   0,   0,   1,   0,   0,   0,   0,   0,   0,   0,   0,
[2022-10-18T13:53:21.283Z]     0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,
[2022-10-18T13:53:21.283Z]     0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,
[2022-10-18T13:53:21.283Z]     0,   0,   0,   0,   0,   0,   0,  49,  86, 120, 244, 149,
[2022-10-18T13:53:21.283Z]    19, 148, 118, 168, 159, 206, 242, 128, 104,  40, 131,  22,
[2022-10-18T13:53:21.283Z]    95, 111,  82, 246, 183,  74, 141,  34,  87,  39,  88, 244,
[2022-10-18T13:53:21.283Z]   133, 165,  93,   0,
[2022-10-18T13:53:21.283Z]   ... 1857 more items
[2022-10-18T13:53:21.283Z] ]
[2022-10-18T13:53:21.283Z] 2022-10-18T13:53:21.279Z @iroha2/client:blocks-stream new block: Enum { tag: 'V1', value: { header: { timestamp: 1666101201212n, consensus_estimation: 2000n, height: 1n, previous_block_hash: [Uint8Array], transactions_hash: [Uint8Array], rejected_transactions_hash: [Uint8Array], genesis_topology: [Enum] }, rejected_transactions: [], transactions: [ [Enum] ], event_recommendations: [], signatures: { signatures: [Map] } } }
[2022-10-18T13:53:21.283Z] 2022-10-18T13:53:21.279Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.277348Z  INFO request{method=POST path=/transaction version=HTTP/1.1 remote.addr=127.0.0.1:55644}: warp::filters::trace: processing request
[2022-10-18T13:53:21.283Z] 2022-10-18T13:53:21.279Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.277615Z  INFO request{method=POST path=/transaction version=HTTP/1.1 remote.addr=127.0.0.1:55644}: warp::filters::trace: finished processing with success status=200
[2022-10-18T13:53:21.539Z] 2022-10-18T13:53:21.289Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.289618Z  INFO request{method=POST path=/transaction version=HTTP/1.1 remote.addr=127.0.0.1:55646}: warp::filters::trace: processing request
[2022-10-18T13:53:21.539Z] 2022-10-18T13:53:21.289Z @iroha2/client:blocks-stream:websocket connection closed; code: 1006, reason: '', was clean: false
[2022-10-18T13:53:21.539Z] 2022-10-18T13:53:21.290Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.289843Z  INFO request{method=POST path=/transaction version=HTTP/1.1 remote.addr=127.0.0.1:55646}: warp::filters::trace: finished processing with success status=200
[2022-10-18T13:53:21.539Z] 2022-10-18T13:53:21.375Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.375896Z  INFO run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: sumeragi Doing block with 2 txs.
[2022-10-18T13:53:21.539Z] 2022-10-18T13:53:21.376Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.376556Z ERROR run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Some events failed to be sent e=channel closed
[2022-10-18T13:53:21.539Z] 2022-10-18T13:53:21.376Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.376573Z ERROR run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Some events failed to be sent e=channel closed
[2022-10-18T13:53:21.539Z] 2022-10-18T13:53:21.376Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.376577Z ERROR run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Some events failed to be sent e=channel closed
[2022-10-18T13:53:21.539Z] 2022-10-18T13:53:21.376Z @iroha2/test-peer:stdout 2022-10-18T13:53:21.376594Z  INFO run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Committing block prev_peer_role=Leader new_peer_role=Leader new_block_height=2 block_hash=d3d161d659abcc6635b3ce0960538eeb029b9406a4ffd8142682471cbd42937e
[2022-10-18T13:53:26.832Z] 2022-10-18T13:53:26.263Z @iroha2/test-peer Killing peer...
[2022-10-18T13:53:26.832Z] 2022-10-18T13:53:26.266Z @iroha2/test-peer:stdout 2022-10-18T13:53:26.265920Z  INFO run{shutdown_receiver=Receiver { inner: Some(Inner { state: State { is_complete: false, is_closed: false, is_rx_task_set: false, is_tx_task_set: false } }) }}: iroha_core::sumeragi::main_loop: Sumeragi Thread is being shutdown shut down.
[2022-10-18T13:53:26.832Z] 2022-10-18T13:53:26.266Z @iroha2/test-peer:stdout 2022-10-18T13:53:26.266813Z  INFO iroha_core::kura: Kura block thread is being shut down
[2022-10-18T13:53:26.832Z] 2022-10-18T13:53:26.266Z @iroha2/test-peer:stdout 2022-10-18T13:53:26.266834Z  INFO iroha_core::kura: Kura block thread is shutting down
[2022-10-18T13:53:26.832Z] 2022-10-18T13:53:26.268Z @iroha2/test-peer Peer exited: [ 0, null ]
[2022-10-18T13:53:26.832Z] 2022-10-18T13:53:26.269Z @iroha2/test-peer Peer is killed
  ```
</details>

### Who can help?

@appetrosyan

### PR DESCRIPTION
**Pull request by Erigara**
### Description of the Change

- Add support for block streaming into `iroha_client` and `iroha_client_cli`;
- Fix iroha block streaming.

### Issue

Closes #2880, #2889.

### Benefits

Fixed bug, block streaming support.

### Possible Drawbacks

None.

### Usage

```
cargo run --bin iroha_client_cli blocks 1
```

### CODE REVIEW

File: client/src/client.rs [hunk 1]
```diff
@@ -867,6 +869,40 @@ impl Client {
...
+    /// - Forwards from [`Self::events_handler`]
+    /// - Forwards from [`blocks_api::BlockIterator::new`]
+    pub fn listen_for_blocks(
+        &self,
+        height: u64,
+    ) -> Result<impl Iterator<Item = Result<VersionedCommittedBlock>>> {
+        blocks_api::BlockIterator::new(self.blocks_handler(height)?)
+    }
+
+    /// Connect (through `WebSocket`) to `async` listen for `Iroha` blocks
```
Comment (outoftardis) [inline client/src/client.rs:884]:
```suggestion
    /// Connect (through `WebSocket`) to `async` to listen for `Iroha` blocks
```

File: client/src/client.rs [hunk 2]
```diff
@@ -1145,184 +1378,132 @@ pub mod events_api {
...
-                match mes {
-                    Ok(WebSocketMessage::Binary(message)) => break handler.message(message)?,
-                    Ok(_) => continue,
-                    Err(err) => return Err(err.into()),
-                }
-            };
-            trace!("`AsyncEventStream` created successfully");
-            Ok(Self { stream, handler })
-        }
+    /// Events API flow. For documentation and example usage please follow to [`crate::http::ws::conn_flow`].
```
Comment (outoftardis) [inline client/src/client.rs:1396]:
```suggestion
    /// Events API flow. For documentation and usage examples, refer to [`crate::http::ws::conn_flow`].
```

File: client/src/client.rs [hunk 3]
```diff
@@ -1145,184 +1378,132 @@ pub mod events_api {
...
+                    msg,
+                    Handshake,
+                )
+            }
         }
-    }
 
-    impl Iterator for EventIterator {
-        type Item = Result<Event>;
+        /// Events API flow handshake handler
```
Comment (outoftardis) [inline client/src/client.rs:1456]:
```suggestion
        /// Handshake handler for Events API flow 
```

File: client/src/client.rs [hunk 4]
```diff
@@ -1145,184 +1378,132 @@ pub mod events_api {
...
-                self.stream.close(None)?;
-                let mes = self.stream.read_message()?;
-                if !mes.is_close() {
-                    return Err(eyre!(
-                        "Server hasn't sent `Close` message for websocket handshake"
-                    ));
-                }
-                Ok(())
-            };
+        /// Events API flow events handler
```
Comment (outoftardis) [inline client/src/client.rs:1478]:
```suggestion
        /// Events handler for Events API flow
```

File: client_cli/src/main.rs [hunk 5]
```diff
@@ -241,6 +243,42 @@ mod events {
...
+mod blocks {
+    use iroha_client::client::Client;
+    use iroha_config::client::Configuration;
+
+    use super::*;
+
+    /// Get event stream from iroha peer
+    #[derive(StructOpt, Debug, Clone, Copy)]
+    pub struct Args {
+        /// Block height to start streaming blocks with
```
Comment (outoftardis) [inline client_cli/src/main.rs:255]:
```suggestion
        ///Block height from which to start streaming blocks
```

File: client/src/client.rs (hunk 1 above)
Comment (Erigara) [reply client/src/client.rs:884]: Rephrased:

```rust
    /// Connect asynchronously (through `WebSocket`) to listen for `Iroha` `pipeline` and `data` events.
```

File: client/src/client.rs [hunk 6]
```diff
@@ -867,6 +869,40 @@ impl Client {
...
+    /// Connect asynchronously (through `WebSocket`) to listen for `Iroha` blocks
+    ///
+    /// # Errors
+    /// - Forwards from [`Self::events_handler`]
+    /// - Forwards from [`blocks_api::BlockIterator::new`]
+    pub async fn listen_for_blocks_async(&self, height: u64) -> Result<AsyncBlockStream> {
+        blocks_api::AsyncBlockStream::new(self.blocks_handler(height)?).await
+    }
+
+    /// Constructs a handler for Blocks API. With this handler you can use any WS client you want.
```
Comment (outoftardis) [inline client/src/client.rs:893]:
```suggestion
    /// Construct a handler for Blocks API. With this handler you can use any WS client you want.
```

File: client/src/client.rs [hunk 7]
```diff
@@ -1145,184 +1378,132 @@ pub mod events_api {
...
-                self.stream.close(None)?;
-                let mes = self.stream.read_message()?;
-                if !mes.is_close() {
-                    return Err(eyre!(
-                        "Server hasn't sent `Close` message for websocket handshake"
-                    ));
-                }
-                Ok(())
-            };
+        /// Blocks API flow events handler
```
Comment (outoftardis) [inline client/src/client.rs:1478]:
```suggestion
        /// Events handler for Blocks API flow
```

**Review by appetrosyan — APPROVED** [review]

File: cli/src/torii/routing.rs [hunk 8]
```diff
@@ -272,15 +273,35 @@ async fn handle_blocks_stream(sumeragi: Arc<Sumeragi>, mut stream: WebSocket) ->
...
+            // This branch sends blocks
+            _ = interval.tick() => {
+                let blocks = sumeragi.blocks_from_height(from_height.try_into().wrap_err("Failed to convert `from_height` into `usize`. You should consider upgrading to a 64-bit arch")?);
+                stream_blocks(&mut from_height, blocks, &mut stream).await?;
+            }
+            // Else branch to prevent panic
+            else => ()
         }
-        stream_blocks(&mut from_height, blocks, &mut stream).await?;
-        std::thread::sleep(std::time::Duration::from_millis(10));
```
Comment (Erigara) [inline cli/src/torii/routing.rs:281]: This statement + loop cause whole iroha to stuck, because of blocking operation in async context.

**Review by SamHSmith — DISMISSED** [review]

File: cli/src/torii/routing.rs (hunk 8 above)
Comment (SamHSmith) [reply cli/src/torii/routing.rs:281]: Yea, good that we are replacing it.

**Review by appetrosyan — APPROVED** [review]

**Review by SamHSmith — APPROVED** [review]

File: client_cli/src/main.rs [hunk 9]
```diff
@@ -104,6 +104,8 @@ pub enum Subcommand {
     Events(events::Args),
     /// The subcommand related to Wasm
     Wasm(wasm::Args),
+    /// The subcommand related to block streaming
```
Comment (outoftardis) [inline client_cli/src/main.rs:107]: note to self: update bash guide here https://hyperledger.github.io/iroha-2-docs/guide/bash.html#_2-configuring-iroha-2

**Pull request merged**